src/main/controllers/helpers/ErrorHelpers.ts

```diff
@@ -362,3 +362,23 @@ export const aboutHearingDocumentsErrors = (req: AppRequest): FormError[] => {
   }
   return errors;
 };
+
+export const agreeingDocumentsForHearingErrors = (req: AppRequest): FormError[] => {
+  const errors: FormError[] = [];
+  if (!req.body.bundlesRespondentAgreedDocWith) {
+    errors.push({ propertyName: 'bundlesRespondentAgreedDocWith', errorType: 'required' });
+  }
+  if (
+    req.body.bundlesRespondentAgreedDocWith === AgreedDocuments.AGREEDBUT &&
```

Comment:
Can probably make the if statements shorter if you get a local variable that holds req.body.bundlesRespondentAgreedDocWith instead of writing it 3 times in length. Not certain if it's much nicer but you can try.

Reply:
Refactored to use a variable

Reply:
Looks great